app/scripts/views/sign_up.js

```diff
@@ -18,7 +19,7 @@ define([
   'views/mixins/resume-token-mixin',
   'views/coppa/coppa-date-picker'
 ],
-function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
+function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck, passwordcheck,
```

Comment:
Instead of including this in the main bundle and causing even more bloat, can you experiment with loading the `passwordcheck` module in `beforeRender` using a nested `require`? We'll have to figure out a cache busting strategy if we do that. I'd like to see how that affects performance.